lib/utils.js

```diff
@@ -0,0 +1,26 @@
+var isString = require('lodash/lang/isString');
+
+function getContainerNode(value) {
+  if (isString(value)) {
+    return document.querySelector(value);
+  }
+  return value;
+}
+
+function renderTemplate(template, data) {
+  var hogan = require('hogan.js');
+  var content;
+
+  if (typeof template === 'string') {
+    content = hogan.compile(template).render(data);
+  } else if (typeof template === 'function') {
+    content = template(data);
+  }
```

Comment:
Oh yes I forgot you are right, you mean throw? If yes then I will add it, we need to be very good at error/reporting/no silent failure from the begining